com.skype.Client.json

```diff
@@ -29,6 +29,11 @@
         "--talk-name=org.kde.StatusNotifierWatcher",
         "--extra-data=skypeforlinux-64.deb:99ed59bfc5521f643071694a23f2387dac016038776d48ab03165c2e35c5c5e4:53937734::https://repo.skype.com/latest/skypeforlinux-64.deb"
     ],
+    "cleanup": ["/include", "/lib/pkgconfig",
```

Comment:
We can do better than generic copy pastes. Some of these are random and don't apply to this.

Reply:
Not sure what the harm is... but if you say so...

Reply:
Flatpak manifests are already copy-paste central and I have to deal with that, by keeping cleanups with their modules there is no global state that can be lost easily, and localized cleanups can be minimal and accurate.

Reply:
Ah, and my point exactly your config-opts have copy paste garbage too...